server.py

```diff
@@ -0,0 +1,100 @@
+"""
+Copyright Schrodinger, LLC. All rights reserved.
+"""
+
+import csv
+import os
+import requests
+import sys
+import tempfile
+from flask import Flask
+from flask import jsonify, request, render_template
+from flask_cors import CORS, cross_origin
+from  rdkit import Chem
+from  rdkit.Chem import rdDepictor
+from functools import partial
+
+
+app = Flask(__name__)
+CORS(app)
+
+sys.path.insert(0, "/software/lib/Darwin-x86_64/rdkit-Release_2018_03_2/lib/python3.6/site-packages/")
```

Comment:
That's interesting that you're able to import rdkit above before you insert this.  Any idea why that works?